.github/workflows/package-publish.yml

```
@@ -0,0 +1,21 @@
+name: package-publish
+
+on: workflow_dispatch
```

yamllint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)